### ISSUE TITLE
training and evalaution 

### ISSUE THREAD

**Issue by golnazhabibi3** [huang-xx/STGAT#12]
Hi, thanks again for sharing your code. My question is about the pretrained model stored in the folder after we train the model and if we can evaluate different datasets (other than the one we trained last time) later. Or there is only one best model  that keeps the latest model trained only? For example, assume  we first train our model for dataset Zara1 and then train our model for dataset ETH, can we evaluate our model on Zara1. Or the pretrained model can be used for ETH only? That is  the pretrained model contains the model it is trained last time, so we can only evaluate ETH, and if we want to evaluate Zara1, we should train the model in Zara1 again? Thanks!

**Comment by golnazhabibi3** [comment]
Another question regarding the error in validation compared to test, I guess the error reported after each epoch (ADE/FDE) is for validation data for that episode, interestingly the error i get fo FDE/ADE on test data is much smaller than the validation error which intuitively does not make sense, do you know why it may happen? thanks

**Comment by huang-xx** [comment]
@golnazhabibi3 We use the leave-one-out approach to measure the performance. To know the performance on one dataset, you need to train a new model. The distribution of the data in the five datasets is quite different, which makes the results obtained on the training set (and validation set) significantly different from the test set, so I did not use the validation set.
